src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -159,7 +180,11 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
           ? undefined
           : {
               quantity: value || 0,
-              units: prev.withdrawnQuantity?.units || accession.remainingQuantity?.units || 'Grams',
+              units:
+                prev.withdrawnQuantity?.units ||
+                (isNurseryTransfer || record.purpose === 'Viability Testing'
+                  ? 'Seeds'
+                  : accession.remainingQuantity?.units || 'Grams'),
```

Comment:
do we need to check if units !== 'Seeds' here just like on line 123?..
wondering if this can be extracted out into a reusable function

Reply:
I don't think so, because the change purpose use effect would have already changed this to grams if units was seeds before. But we can add it just in case

Reply:
I will merge it like this, we are revisiting this later anyway

Reply:
🎉